odoo/http.py

```diff
@@ -542,6 +542,15 @@ def from_attachment(cls, attachment):
             self.last_modified = attachment.write_date
             self.size = len(self.data)
 
+        elif (attachment.type == 'cloud_storage' and
+              attachment.env['res.config.settings']._get_cloud_storage_configuration()):
```

Comment:
Huh.  This is unmodular.  Woudn't it be possible to delegate that stuff to the attachment record ?
The hole method could look like:
```py
self = cls(...)
attachment._to_stream(self)
return self
```